charts/drax/values.yaml

```
@@ -1462,7 +1462,16 @@ promtail-notifications:
                   stndDefinedNamespace: event.commonEventHeader.stndDefinedNamespace
                   version: event.commonEventHeader.version
                   timeZoneOffset: event.commonEventHeader.timeZoneOffset
+                  notificationType: event.stndDefinedFields.data.notificationType
                   vesEventListenerVersion: event.commonEventHeader.vesEventListenerVersion
+            - regex:
+               expression: '^(?P<keep>notifyEvent)$'
+               source: notificationType
+            - labels:
+               keep: '{{.keep}}'
```

Promtail will use the capture group name by default if left empty:

```suggestion
               keep:
```